### PR TITLE
Rename Services to Actions in integrations: F

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

SSIA, adjust all occurrences for integrations starting with the letter F

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Updated terminology from "services" to "actions" for fan control functionalities, aligning with the Home Assistant context.
  - Clarified usage of the `speed_count` parameter for fan speed adjustments.
  - Revised control commands for managing FFmpeg processes.
  - Updated instructions for safe filter replacement in Flexit units.
  - Enhanced permission settings and authorization process for Freebox integration.

### CODE REVIEW

File: source/_integrations/fan.markdown [hunk 1]
```diff
@@ -15,24 +15,24 @@ The fan integration allows you to control and monitor fan devices.
...
+### Fan control actions
 
-Available services:
+Available actions:
 `fan.set_percentage`, `fan.set_preset_mode`, `fan.set_direction`, `fan.oscillate`, `fan.turn_on`, `fan.turn_off`, `fan.toggle`, `fan.increase_speed`, `fan.decrease_speed`
 
 {% note %}
 
-Not all fan services may be available for your platform. You can check which services are available for your fan(s) under **Developer Tools** > **Services**.
+Not all fan actions may be available for your platform. You can check which action are available for your fan(s) under **Developer Tools** > **Actions**.
```
Comment (mib1185) [inline source/_integrations/fan.markdown]:
```suggestion
Not all fan actions may be available for your platform. You can check which actions are available for your fan(s) under **Developer Tools** > **Actions**.
```

File: source/_integrations/fan.template.markdown [hunk 2]
```diff
@@ -154,7 +154,7 @@ When converting a fan with 3 speeds from the old fan entity model, the following
 ### Helper fan
 
 This example uses an input_boolean and an input_number to mimic a fan, and 
-the example shows multiple service calls for set_percentage.  
+the example shows multiple action for `set_percentage`.
```
Comment (mib1185) [inline source/_integrations/fan.template.markdown]:
```suggestion
the example shows multiple actions for `set_percentage`.
```

File: source/_integrations/ffmpeg_motion.markdown [hunk 3]
```diff
@@ -18,7 +18,7 @@ The `ffmpeg` {% term integration %} allows you to use any video feed with [FFmpe
 
 {% note %}
 
-If the `ffmpeg` process is broken, the sensor will be unavailable. To control the FFmpeg process of sensor, use the service `ffmpeg.start`, `ffmpeg.stop`, `ffmpeg.restart`.
+If the `ffmpeg` process is broken, the sensor will be unavailable. To control the FFmpeg process of the sensor, use the `ffmpeg.start`, `ffmpeg.stop`, and `ffmpeg.restart` actions.
```
Comment (mib1185) [inline source/_integrations/ffmpeg_motion.markdown:21]:
```suggestion
If the `ffmpeg` process is broken, the sensor will be unavailable. To control the FFmpeg process of the sensor, use the `ffmpeg.start`, `ffmpeg.stop` and `ffmpeg.restart` actions.
```

File: source/_integrations/ffmpeg_noise.markdown [hunk 4]
```diff
@@ -18,7 +18,7 @@ The `ffmpeg` {% term integration %} allows you to use any video or audio feed wi
 
 {% note %}
 
-If the `ffmpeg` process is broken, the sensor will be unavailable. To control the FFmpeg process of sensor, use the service `ffmpeg.start`, `ffmpeg.stop`, `ffmpeg.restart`.
+If the `ffmpeg` process is broken, the sensor will be unavailable. To control the FFmpeg process of sensor, use the `ffmpeg.start`, `ffmpeg.stop`, and `ffmpeg.restart` actions.
```
Comment (mib1185) [inline source/_integrations/ffmpeg_noise.markdown:21]:
```suggestion
If the `ffmpeg` process is broken, the sensor will be unavailable. To control the FFmpeg process of sensor, use the `ffmpeg.start`, `ffmpeg.stop` and `ffmpeg.restart` actions.
```

**Review by mib1185 — CHANGES_REQUESTED** [review]
just some small comments from my side

File: source/_integrations/ffmpeg_motion.markdown (hunk 3 above)
Comment (frenck) [reply source/_integrations/ffmpeg_motion.markdown:21]: This suggestion is not correct, it should have a comma here.

File: source/_integrations/ffmpeg_noise.markdown (hunk 4 above)
Comment (frenck) [reply source/_integrations/ffmpeg_noise.markdown:21]: same.

File: source/_integrations/ffmpeg_motion.markdown (hunk 3 above)
Comment (mib1185) [reply source/_integrations/ffmpeg_motion.markdown:21]: now I learned about the so called "Oxford Comma" ... it is still looks weird to me :see_no_evil: but i'm not a native English :grin:

**Review by mib1185 — APPROVED** [review]
LGTM, thx @frenck :+1:

**Pull request merged**